File: Iniciante/JavaScript/1036.js
// When executing the code locally Remeber to:
//  -> remove the . before /dev and 'txt' after stdin when submitting to beecrowd (or when uploading to github, but that is optional)
var input = require('fs').readFileSync('./dev/stdin.txt', 'utf8');
var lines = input.trim().split(' ');

const [a,b,c] = [...lines.map(Number)];

let delta = (b**2) - 4*a*c;

if( a === 0 || delta < 0){
    console.log('Impossivel calcular');
}else{
    let r1 = (-Math.abs(b) + Math.sqrt(delta))/(2*a);
    let r2 = (-Math.abs(b) - Math.sqrt(delta))/(2*a);
    console.log(`R1 = ${r1.toFixed(5)}\nR2 = ${r2.toFixed(5)}`);
}

File: Iniciante/Python/1036.py
import math

a,b,c = map(float, input().split())

delta = (b**2) - 4*a*c

if a == 0 or delta < 0:
    print("Impossivel calcular")
else:
    r1 = (-abs(b) + math.sqrt(delta))/(2*a)
    r2 = (-abs(b) - math.sqrt(delta))/(2*a)
    print(f"R1 = {r1:.5f}\nR2 = {r2:.5f}")